### PR TITLE
Modify the configuration file structure and add parameter fields for subscription events

### PR DESCRIPTION
**Pull request by yaofighting**
Modify the configuration file structure and add parameter fields for subscription events

Signed-off-by: yaofighting <siyao@zju.edu.cn>

### Motivation
In some scenarios, only specifying the type of subscription event cannot meet our needs. Therefore, we need to enhance the features of the subscription part, support more detailed parameter lists, and control the CPP side to send only the event information we are concerned about to the Go side. For example, we need to specify system calls that filter latency is less than 500ms. When subscribing to events, we tell the CPP side to send only data that meets this condition, so as to avoid performance loss caused by data filtering on the Go side.
## Related Issue
#366

### CODE REVIEW

**Review by sanyangji — DISMISSED** [review]
LGTM.
We may unify the Naming Convention for cpp codes and cgo codes.
 @dxsup @jundizhou

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@
 - 
 - Support the protocol RocketMQ.([#328](https://github.com/KindlingProject/kindling/pull/328))
 - Add a new tool: A debug tool for Trace Profiling is provided for developers to troubleshoot problems.([#363](https://github.com/CloudDectective-Harmonycloud/kindling/pull/363))
+- Modify the configuration file structure and add parameter fields for subscription events. ([#368](https://github.com/CloudDectective-Harmonycloud/kindling/pull/368))
```
Comment (dxsup) [inline CHANGELOG.md:11]: I consider it more like an "Enhancement" than a "New feature".

**Comment by dxsup** [comment]
> LGTM.
> We may unify the Naming Convention for cpp codes and cgo codes.
> @dxsup @jundizhou

For example? I didn't see any difference on "Naming Convention" between these files. Do you mean the indentation style?

File: CHANGELOG.md (hunk 1 above)
Comment (yaofighting) [reply CHANGELOG.md]: I think so,  I will modify it immediately.

**Comment by sanyangji** [comment]
> > LGTM.
> > We may unify the Naming Convention for cpp codes and cgo codes.
> > @dxsup @jundizhou
> 
> For example? I didn't see any difference on "Naming Convention" between these files. Do you mean the indentation style?

``

> > LGTM.
> > We may unify the Naming Convention for cpp codes and cgo codes.
> > @dxsup @jundizhou
> 
> For example? I didn't see any difference on "Naming Convention" between these files. Do you mean the indentation style?
`void sub_event(char* eventName, char* category, event_params_for_subscribe params[])`

**Comment by sanyangji** [comment]
> > LGTM.
> > We may unify the Naming Convention for cpp codes and cgo codes.
> > @dxsup @jundizhou
> 
> For example? I didn't see any difference on "Naming Convention" between these files. Do you mean the indentation style?

`void sub_event(char* eventName, char* category, event_params_for_subscribe params[])` Like In this line, eventName is different from others

**Comment by dxsup** [comment]
OK, so first let's determine the convention on naming, then we can rename them all.

**Review by sanyangji — APPROVED** [review]

**Comment by dxsup** [comment]
We can open another PR for the "naming convention". Back to this one, LGTM.

**Review by dxsup — APPROVED** [review]

**Comment by yaofighting** [comment]
I think we should use _underscore_case_ in C/C++ and _camelCase_ in Golang.

**Comment by yaofighting** [comment]
I think we should use _underscore_case_ in C/C++ and _camelCase_ in Golang.

**Review by dxsup — APPROVED** [review]

**Pull request merged**